designer/client/page-edit.js

```diff
@@ -140,19 +136,13 @@ class PageEdit extends React.Component {
         </div>
 
         <div className='govuk-form-group'>
-          <label className='govuk-label govuk-label--s' htmlFor='page-path'>Path</label>
-          <input className='govuk-input' id='page-path' name='path'
-            type='text' defaultValue={page.path}
-            onChange={e => e.target.setCustomValidity('')} />
-        </div>
-
-        <div className='govuk-form-group'>
-          <label className='govuk-label govuk-label--s' htmlFor='page-title'>Title (optional)</label>
+          <label className='govuk-label govuk-label--s' htmlFor='page-title'>Title</label>
           <span id='page-title-hint' className='govuk-hint'>
             If not supplied, the title of the first question will be used.
           </span>
-          <input className='govuk-input' id='page-title' name='title'
-            type='text' defaultValue={page.title} aria-describedby='page-title-hint' />
+          <input className='govuk-input' id='page-title' name='title' type='text' defaultValue={page.title}
+            aria-describedby='page-title-hint' required
+            onChange={e => e.target.setCustomValidity('')} />
         </div>
 
         <div className='govuk-form-group'>
```

Comment:
Can we merge these - why do we need a create and an edit? Aren't they doing the same thing?

Reply:
I think it's there mainly for the default value so it is visible when you click to edit...otherwise when you edit something it won't display it after you click save and press edit again. Maybe it can still be merged not sure.